File: app/src/main/java/com/sendbird/calls/quickstart/utils/ToastUtils.java
package com.sendbird.calls.quickstart.utils;

import android.content.Context;
import android.text.TextUtils;
import android.widget.Toast;

public class ToastUtils {

    public static void showToast(Context context, String text) {
        if (context != null && !TextUtils.isEmpty(text)) {
            Toast.makeText(context, text, Toast.LENGTH_LONG).show();
        }
    }
}

File: directcall-kotlin/app/src/main/java/com/sendbird/calls/quickstart/utils/ToastUtils.kt
package com.sendbird.calls.quickstart.utils

import android.content.Context
import android.widget.Toast
import androidx.fragment.app.Fragment

fun Context.showToast(text: String) {
    Toast.makeText(this, text, Toast.LENGTH_LONG).show()
}

fun Fragment.showToast(text: String) {
    requireContext().showToast(text)
}
